tests/parsers/amcache.py

```diff
@@ -8,14 +8,12 @@
 
 from plaso.parsers import amcache
 
-from tests import test_lib as shared_test_lib
 from tests.parsers import test_lib
 
 
 class AmcacheParserTest(test_lib.ParserTestCase):
   """Tests for the Amcache Registry plugin."""
 
-  @shared_test_lib.skipUnlessHasTestFile(['Amcache.hve'])
```

Comment:
Shouldnt there be a
```
test_file_path = self._GetTestFilePath(['Amcache.hve')])
self._SkipIfPathNotExists(test_file_path)

```

Reply:
Line 23 calls `storage_writer = self._ParseFile(['Amcache.hve'], parser)` which includes a call to `_SkipIfPathNotExists`